packages/nouns-webapp/src/components/Winner/index.tsx

```diff
@@ -1,30 +1,58 @@
-import { Button } from 'react-bootstrap';
+import { Button, Row, Container, Col } from 'react-bootstrap';
 import { useAppSelector } from '../../hooks';
 import classes from './Winner.module.css';
 import ShortAddress from '../ShortAddress';
 import { Link } from 'react-router-dom';
 
-const Winner: React.FC<{ winner: string }> = props => {
-  const { winner } = props;
+interface WinnerProps {
+  winner: string;
+  isNounders?: boolean;
+}
+
+const Winner: React.FC<WinnerProps> = props => {
+  const { winner, isNounders } = props;
   const activeAccount = useAppSelector(state => state.account.activeAccount);
 
+  const nonNounderNounContent =
```

Comment:
Let's align the verification button to the right of the copy ("You!"). 

Would be cool to add a little design love here since it's the first instance of a bidder being recognized. Maybe worthy of an issue?

<img width="555" alt="Screen Shot 2022-01-28 at 5 49 19 PM" src="https://user-images.githubusercontent.com/85328329/151632388-f5b11901-0e57-4d29-b4ff-2dfc60b5d8c6.png">

Reply:
Created an issue: 

https://github.com/nounsDAO/nouns-monorepo/issues/342

Will default to punting this from this PR unless told otherwise